EXILED/Exiled.API/Features/Roles/FpcRole.cs

```diff
@@ -243,6 +243,42 @@ public bool IsNoclipEnabled
             set => Owner.ReferenceHub.playerStats.GetModule<AdminFlagsStat>().SetFlag(AdminFlags.Noclip, value);
         }
 
+        /// <summary>
+        /// Gets or sets <see cref="API.Features.Ragdoll"/> for this role.
+        /// </summary>
+        public Ragdoll Ragdoll
+        {
+            get => Ragdoll.Get(FirstPersonController.Ragdoll);
+            set => FirstPersonController.Ragdoll = value.Base;
+        }
+
+        /// <summary>
+        /// Gets or sets voice module for this role.
+        /// </summary>
+        public VoiceModuleBase VoiceModule
+        {
+            get => FirstPersonController.VoiceModule;
+            set => FirstPersonController.VoiceModule = value;
+        }
```

Comment:
`VoiceModule` not a direct correlation with `FpcRole`, spectators and 079 have that too

actually we already have that in `Player` class, so it makes little sense at all

(but if you implement voicemodule based on `IVoiceRole`, like we have `ISubroutinedScpRole` in exiled, it will be even better than now)